arangod/Agency/Agent.cpp

```diff
@@ -44,6 +44,12 @@ using namespace std::chrono;
 namespace arangodb {
 namespace consensus {
 
+// Instanciations of some declarations in AgencyCommon.h:
+
+std::string const pubApiPrefix("/_api/agency/");
+std::string const privApiPrefix("/_api/agency_priv/");
+std::string const NO_LEADER("");
```

Comment:
Looking at messages in followNoLock().  I think there is a code path where there is no leader and we log "following  in term X" because NO_LEADER has no text string (line 224 in constituency.cpp).  I think I have seen this in live system.  Maybe this could be "<no one>".

Reply:
Added quotes in various places.